force-app/main/default/classes/NextStepSchedule.cls

```diff
@@ -19,16 +19,20 @@
  * 
  * Note: To make full use of this class, ensure that the NextStepBatch class is implemented and operational.
  */
-public with sharing class NextStepSchedule {
+public with sharing class NextStepSchedule implements Schedulable {
 
-    /*
+     /*
      * Update/Uncomment the template code to create a method
      * Execute the batch job NextStepBatch using the Database.executeBatch method
      * Configure and schedule the job using the Apex Scheduler in Salesforce Setup
      * Code to execute the batch job should not need to change other than uncommenting the code
      */
-    //[public | private ] [static] data_type execute (input parameters) {
-        //NextStepBatch batch = new NextStepBatch(); 
-        //Database.executeBatch(batch);
-    //}
+
+     public void execute(SchedulableContext sc) {
+        System.debug('Schedule Started');
+        NextStepBatch batch = new NextStepBatch(); 
+        Database.executeBatch(batch);
+
```

Comment:
As a best practice try to remove System.debug  statements

If you want you can save a line of code with   
  Database.executeBatch(new NextStepBatch());